Python/libraries/recognizers-date-time/recognizers_date_time/date_time/portuguese/parsers.py

```diff
@@ -54,7 +54,7 @@ def parse_specific_time_of_day(self, source: str, reference: datetime) -> DateTi
 
         # handle Date followed by morning, afternoon
         # Add handling code to handle morning, afternoon followed by Date
-        # Add handling code to handle early/late morning, afternoon
+        # Add handling code to handle early/late morning, afternoon.
```

Comment:
No need for such commits. We can either re-run the same build. Or close/reopen the PR and a new build is triggered.